ls1intum/Artemis
# `Assessments`: Display feedback from structured grading criteria more clearly

### Checklist
#### General


#### Client

### Motivation and Context



### Description

Highlighting feedback generated through structured grading criteria so that it is more visible to tutors when assessing text exercises. 

### Steps for Testing

Prerequisites:
- 1 Tutor
- 1 Text Exercise with structured grading criteria

1. Navigate to the `Exercise Dashboard` of the text exercise.
2. Start an assessment of a submission.
3. Give feedback using structured grading criteria.
4. Check that _`Feedback:`_ ends with a colon, followed by the feedback text displayed as bold.
5. Check that the question mark icon is at the end. 

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Updated version:

<img width="914" alt="Bildschirmfoto 2021-12-23 um 22 46 08" src="https://user-images.githubusercontent.com/73841445/147293922-0770c93b-74c7-44d4-8419-da3fee2920fa.png">


Old version:

<img width="915" alt="Bildschirmfoto 2021-12-23 um 16 33 28" src="https://user-images.githubusercontent.com/73841445/147293948-eae9e9e4-c79b-495e-87a7-d383c023b16a.png">
